ppr-ui/src/components/tables/RegistrationTable.vue

```diff
@@ -484,12 +486,15 @@ export default defineComponent({
       return false
     }
 
-    const isNewRegParentItem = (item: RegistrationSummaryIF): boolean => {
+    const isNewRegParentItem = (item: RegistrationSummaryIF | MhRegistrationSummaryIF): boolean => {
+      const regItem = item as RegistrationSummaryIF
+      const mhRegItem = item as MhRegistrationSummaryIF
       if (item.expand === undefined && item.changes !== undefined) item.expand = false
+      const registrationNumber = regItem.registrationNumber || mhRegItem.mhrNumber
       return (
         !!localState.newReg?.addedRegParent &&
         localState.newReg.addedRegParent !== '' &&
-        localState.newReg.addedRegParent === item.registrationNumber
```

Comment:
What was wrong with the original logic? If `item` of both RegistrationSummaryIF or RegistrationSummaryIF types has **mhrNumber** then maybe we can keep it that way?

Reply:
Another good question... The reason why I did this is because the original logic used the RegistrationSummaryIF and on the first iteration of the mhr table this logic was not required.  Mhr registrations use MhRegistrationSummaryIF).  It does not contain registration number.... Unfortunately there is no common interface for the grid...It would be too big of a change... So this method now takes RegistrationSummaryIF  or MhRegistrationSummaryIF.  see line 490 to 493..

     const regItem = item as RegistrationSummaryIF
      const mhRegItem = item as MhRegistrationSummaryIF
      if (item.expand === undefined && item.changes !== undefined) item.expand = false
      **const registrationNumber = regItem.registrationNumber || mhRegItem.mhrNumber**

Reply:
I see. I think we could remove lines 490 and 491, and then just rewrite 493 to:
`const registrationNumber = item.registrationNumber || item.mhrNumber`